### DISCUSSION TITLE
giscusSolidity

### DISCUSSION THREAD

**Discussion by giscus** [web3study/web3learning#66] [General]
# giscusSolidity

主要写一些关于区块链的基本知识，去中心化、比特币，简单的概括，如果需要详情，可以跳转到对应的说明文档

http://localhost:3000/wiki/solidity/

**Comment by yzbbanban** [comment]
||ヽ(*￣▽￣*)ノミ|Ю

**Comment by yzbbanban** [comment]
单独挖多哇
